packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1881,6 +1953,14 @@ export class ContainerRuntime
 				case ContainerMessageType.BlobAttach:
 					this.blobManager.processBlobAttachOp(message, local);
 					break;
+				case ContainerMessageType.IdAllocation:
+					if (this.idCompressor === undefined) {
+						throw new Error(
+							"Received an IdAllocation op without having the compressor enabled",
+						);
+					}
+					this.idCompressor?.finalizeCreationRange(message.contents as IdCreationRange);
```

Comment:
nit: I believe the ? is no longer required since you have an undefined check above.